### PR TITLE
Mobile version portfolio

### PR DESCRIPTION
**Pull request by bamsi**
Hello, in this PR I:
-add linters
-add toolbar
-add top header section
-add descriptive read me file

### CODE REVIEW

**Review by JuliCarracedo — CHANGES_REQUESTED** [review]
Hi @bamsi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [✔️](https://emojipedia.org/check-mark/) Great and very professional PR and README
- [✔️](https://emojipedia.org/check-mark/) Great job applying the requirements and the figma template!

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  </head>
+  <body>
+    <main class="main-container">
+      <header class="toolbar">
+        <h1>Welcome</h1>
+        <span> <img src="images/toggle.png" alt="toggle icon" /> </span>
+      </header>
+      <section class="top-header">
+        <div class="header-title">
+          <h2>Hey there. I'm Bamsi</h2>
```
Comment (JuliCarracedo) [inline index.html]: - [ ] Kindly make this an `h1` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <main class="main-container">
+      <header class="toolbar">
+        <h1>Welcome</h1>
```
Comment (JuliCarracedo) [inline index.html]: - [ ] Kindly make this an `a` tag. Normally, logos are meant to work as links to the homepage.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        <div class="header-title">
+          <h2>Hey there. I'm Bamsi</h2>
+          <h2 class="color-green">I'm a software developer</h2>
+        </div>
+        <div class="header-body">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
```
Comment (JuliCarracedo) [inline index.html:26]: - [ ] Kindly adjust the color of this section to match the Figma template
review_image1

**Comment by bamsi** [comment]
Hi @JuliCarracedo 
Thank you for your comments, I have fixed all the issues. can you please review it?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <main class="main-container">
+      <header class="toolbar">
+        <a>Welcome</a>
+        <span> <img src="images/toggle.png" alt="toggle icon" /> </span>
```
Comment (MeqdamAlqudah) [inline index.html]: - [ Optional] good job by changing the logo to a link but it would be better if you also if the link send the user to the home page

Comment (MeqdamAlqudah) [inline index.html]: - [x] Please fix the position of the logo 
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+            </li>
+            <li>
+              <a href="#"><img src="images/m.png" alt="m icon" /></a>
+            </li>
+          </ul>
+        </div>
+
+        <div class="down-arrow">
+          <img src="images/arrow.png" alt="arrow" />
+        </div>
```
Comment (MeqdamAlqudah) [inline index.html:52]: - [ ] Please fix the position of the down arrow and make it match [Figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
- [ ] 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @bamsi , 

Good job so far!

- [ ] Major highlights

- The PR summary and commit messages are very descriptive
- Passed linters checks

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (bamsi) [reply index.html]: I dont understand this, can you elaborate

**Review by Kingstalux — APPROVED** [review]
Hi @bamsi,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966

Great job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**